crosswalks/usgs-modelcatalog.csv

```diff
@@ -0,0 +1,74 @@
+Property,usgs-modelcatalog
+codeRepository,software.usgs_software_gitlab_release_urls / software.external_software_repository_urls
```

Comment:
`usgs_software_gitlab_release_urls` sounds like it would point to tarballs/zipballs instead of a VCS repository. Isn't `installUrl` a more relevant mapping for this term? Though examples in https://github.com/DOI-USGS/modelcatalog-tam/blob/6582b097d27a1fd461131cf0c74f19937f7abd4a/tests/test_basicmetadata.py look like it's webpages. 

`external_software_repository_urls` also seems to point to webpages in practice, eg. https://github.com/DOI-USGS/modelcatalog-tam/blob/6582b097d27a1fd461131cf0c74f19937f7abd4a/tests/test_basicmetadata.py#L124

Reply:
`usgs_software_gitlab_release_urls` and `external_software_repository_urls` are meant to point to the top page of a public git repository. The confusion may stem from the use of "release" to mean a repository that has gone through proper review and approval processes at the USGS. The published software projects are called "software releases." At the current time, the official release must be on gitlab. However, there are also github repos allowed ("external" because they are not official USGS releases). 
Summary: Those terms both point to webpages/repositories, and not tarballs or zipballs.